packages/terra-base/src/Base.scss

```diff
@@ -1,6 +1,8 @@
-// Normalize box-sizing.
-// More info: https://css-tricks.com/box-sizing/#article-header-id-6
 html {
+  background-color: var(--terra-base-background-color, #fff);
+  background-image: var(--terra-base-background-image, none);
```

Comment:
It's probably unlikely, but are there any concerns about this move being non-passive? It would probably require a really awkward CSS setup to cause any issues, but it might be able to cause a users workaround background color to change to white, or their image to change to none if they had one defined in a roundabout way.

Reply:
I don't see it as a concern, we can't predict customizations outside of our components.

Reply:
That's fair, Base is tricky because it has no component scope just global impact